### ISSUE TITLE
Usages of public classes inside other classes are not updated correctly

### ISSUE THREAD

**Issue by BlueManiac** [lsoft/AdjustNamespace#2]
Example from

file1 inside folder Test2:
```csharp
namespace Test
{
    public class TestClass
    {
        public class InnerTestClass
        {
        }
    }
}
```
file2
```csharp
using Test;

namespace ConsoleApp1
{
    public class TestStuff
    {
        public TestStuff(TestClass.InnerTestClass p)
        {

        }
    }
}
```

Result file 2
```csharp
namespace ConsoleApp1
{
    public class TestStuff
    {
        public TestStuff(ConsoleApp1.Test2.InnerTestClass p)
        {

        }
    }
}
```
Expected
```csharp
using ConsoleApp1.Test2;

namespace ConsoleApp1
{
    public class TestStuff
    {
        public TestStuff(TestClass.InnerTestClass p)
        {

        }
    }
}
```

Otherwise it has worked great! Thanks for making this :)

**Comment by lsoft** [comment]
Interesting! Thanks!

**Comment by lsoft** [comment]
Too late is better than never, yeah? 😁 Fixed in 0.1.10.
